docs/getting-started/development-environment.md

```diff
@@ -88,3 +88,14 @@ yarn create-plugin # Create a new plugin
 > See
 > [package.json](https://github.com/spotify/backstage/blob/master/package.json)
 > for other yarn commands/options.
+
+## Local configuration
+
+Backstage allows you to specify the configuration used while running the
+application on your computer. Local configuration is read from
```

Comment:
Maybe mention that it merges with the others? So people don't copy over all of it

Reply:
Added a message about override in https://github.com/spotify/backstage/pull/2710/commits/65a06ce1e8c5f7c807bc8166b9541850c58697c0